src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -48,6 +48,10 @@
   --pf-c-dropdown__menu-item--disabled--Color: var(--pf-global--Color--dark-200);
   --pf-c-dropdown__menu-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
 
+  // Complementary Text
```

Comment:
This doesn't seem like a part of a dropdown, just some special content that went into the dropdown item. Do we anticipate that other kinds of dropdowns would include complementary text, or is this unique to pagination? If it belongs in dropdown, then it should be documented and shown in a dropdown example.